docs/apps/srt-test-multiplex.md

```diff
@@ -1,26 +1,27 @@
-## srt-multiplex
+## srt-test-multiplex
```

Comment:
Something is wrong with the level of headings in this document. It should start with "# srt-test-multiplex" and then "## Usage", etc.. Could you please go through the document and update the levels carefully?